### ISSUE TITLE
DistSeedRepeatError when running many sims

### ISSUE THREAD

**Issue by daniel-klein** [starsimhub/starsim#534]
When running 4,000 simulations (TBsim), I occasionally trigger a DistSeedRepeatError:
```python
DistSeedRepeatError ss.bernoulli(people_female_default, pars={'p': 0.5}) ss.random(people__states_13459533552_default, pars={})

starsim.distributions.DistSeedRepeatError: A common seed was found between ss.bernoulli(people_female_default, pars={'p': 0.5}) and ss.random(people__states_13459533552_default, pars={}). This is likely caused by incorrect initialization of the parent Dists object.
```

I don't believe this message is caused by incorrect initialization, because the error is triggered only rarely.

**Comment by daniel-klein** [comment]
As with #533, this appears to be related to:
```python
ss.FloatArr('ppf_LS_to_presymp', default=ss.random())
```
wherein a FloatArr is to be initialized to a Dist. It's not working, and it also appears as though this might be the cause of the DistSeedRepeatError.

**Comment by cliffckerr** [comment]
Closed by #536
